# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lcap/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lcap-0.0.6/Makefile.vanilla	2007-06-17 13:26:43.031213312 +0200
+++ lcap-0.0.6/Makefile	2007-06-17 13:27:10.048752954 +0200
@@ -29,14 +29,16 @@
 LDFLAGS = -s
 
 FILES = COPYING INSTALL Makefile README lcap.c lcap.spec lcap.8
+
+prefix=/usr
   
 all:		lcap
 
 lcap:		lcap.o
 
 install:	lcap
-		install -m700 -D lcap $(prefix)/sbin/lcap
-		install -m444 -D lcap.8 $(prefix)/usr/man/man8/lcap.8
+		install -m700 -D lcap $(DESTDIR)/sbin/lcap
+		install -m444 -D lcap.8 $(DESTDIR)$(prefix)/man/man8/lcap.8
 
 rpms:
 		$(MAKE) dist
